### PR TITLE
Bump docker images for 0.25.0

### PR DESCRIPTION
**Pull request by danielhoherd**
This bumps a bunch of docker images for the 0.25.0 release. There are a LOT of changes in here and I have done no testing of any of these images.

Releated to https://github.com/astronomer/issues/issues/2749

### CODE REVIEW

**Review by vishwas-astro — COMMENTED** [review]
How about kibana, pgbouncer-exporter and elasticsearch itself?

File: charts/nginx/values.yaml [hunk 1]
```diff
@@ -9,11 +9,11 @@ tolerations: []
 images:
   nginx:
     repository: quay.io/astronomer/ap-nginx
-    tag: 0.43.0
+    tag: 0.45.0
     pullPolicy: IfNotPresent
   defaultBackend:
     repository: quay.io/astronomer/ap-default-backend
-    tag: 0.16.1
+    tag: 0.15.0
```
Comment (danielhoherd) [inline charts/nginx/values.yaml:16]:
```suggestion
    tag: 0.25.1
```

[the remark this answers is not in the document]
Comment (danielhoherd) [reply charts/nginx/values.yaml]: Good catch!

File: charts/elasticsearch/values.yaml [hunk 2]
```diff
@@ -14,19 +14,19 @@ images:
     pullPolicy: IfNotPresent
   init:
     repository: quay.io/astronomer/ap-base
-    tag: 3.13.2
+    tag: 3.13.5
```
Comment (vishwas-astro) [inline charts/elasticsearch/values.yaml:17]: you should probably update tag in line 13 to 7.10.2-1

**Comment by danielhoherd** [comment]
pgbouncer-exporter is part of airflow-chart.

Bumped kibana and elasticsearch.

**Comment by vishwas-astro** [comment]
elasticsearch needs to be bumped to 7.10.2-1 and
kibana needs to be bumped to 7.10.2-1 as well.

Rest all looks good.

**Review by vishwas-astro — APPROVED** [review]

**Pull request merged**